packages/nodes-base/nodes/Google/Sheet/v2/actions/sheet/read.operation.ts

```diff
@@ -235,14 +235,14 @@ export async function execute(
 				| 'AND'
 				| 'OR';
 
-			responseData = await sheet.lookupValues(
-				data as string[][],
-				headerRow,
-				firstDataRow,
+			responseData = await sheet.lookupValues({
+				inputData: data as string[][],
+				keyRowIndex: headerRow,
+				dataStartRowIndex: firstDataRow,
```

Comment:
same as before could you add names for the rest of the parameters to make the code more consistent and readable?